packages/dev-tools-pages/webpack.config.js

```diff
@@ -60,21 +75,35 @@ const config = {
     devServer: {
         port: 3572,
         disableHostCheck: true,
+        overlay: true,
+        historyApiFallback: true,
     },
 };
 
 module.exports = (_env, argv) => {
-    let plugins = [];
+    let plugins = [
+        new CleanWebpackPlugin('public'),
+        ...pages.map(p => new HtmlWebpackPlugin(p)),
+        new CopyWebpackPlugin([
+            { from: 'assets/crawl.html', to: 'index.html' },
+            { from: 'assets/fonts', to: 'fonts' },
+            { from: 'assets/images', to: 'images' },
+        ]),
+    ];
     if (argv.mode === 'development') {
         config.mode = 'development';
     } else {
         config.mode = 'production';
+        config.output.filename = 'bundle-[name].[chunkhash].js';
+        config.output.chunkFilename = 'bundle-[name].[chunkhash].js';
+
         plugins = plugins.concat([
             new webpack.DefinePlugin({
                 'process.env': {
-                    NODE_ENV: JSON.stringify(process.env.NODE_ENV),
+                    NODE_ENV: JSON.stringify(process.env.NODE_ENV || config.mode),
                 },
             }),
+            //new BundleAnalyzerPlugin(),
```

Comment:
Comment?

Reply:
Left it in there to check the bundle, do you want me to remove it?

Reply:
Maybe just add a comment explaining that it's there on purpose.